tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/document/documents.controller.js

```diff
@@ -1832,7 +1832,7 @@
         vm.getDocumentAssets = function (row) {
             vm.assetsuccess = false;
             var Client = {
-                Id: (row.entity.documentParentUrl + "/" + row.entity.documentName + "." + row.entity.documentExtension).replace(configs.uri.SPOsiteURL, ""),
+                Id: (row.entity.documentParentUrl.replace("/Forms/AllItems.aspx", "") + "/" +row.entity.documentName + "." +row.entity.documentExtension).replace(configs.uri.SPOsiteURL, ""),

```

Comment:
docGuid was coming as empty and document details page was giving error. To fix that error this code was written